### PR TITLE
Fix Publish Profile & Update Dependencies

### PR DESCRIPTION
**Pull request by bennettforkner**
This PR updates the SMB path to the 360api server in the publish profile so that one-click publishing works properly.

It also updates a series of vulnerable dependencies to their oldest LTS version. I was not looking to cause breaking changes, rather to just pass the threshold of security.

### CODE REVIEW

File: Phoenix/Phoenix.csproj [hunk 1]
```diff
@@ -254,6 +253,11 @@
     <None Include="Properties\PublishProfiles\RCIPublish.pubxml">
       <SubType>Designer</SubType>
     </None>
+    <None Include="Scripts\jquery.validate-vsdoc.js" />
+    <Content Include="Scripts\jquery.validate.js" />
+    <Content Include="Scripts\jquery.validate.min.js" />
+    <Content Include="Scripts\jquery.validate.unobtrusive.js" />
+    <Content Include="Scripts\jquery.validate.unobtrusive.min.js" />
```
Comment (EjPlatzer) [inline Phoenix/Phoenix.csproj:260]: Do you know why these were added?

File: Phoenix/Web.config [hunk 2]
```diff
@@ -67,7 +67,7 @@
   </entityFramework>
   <connectionStrings>
     <add name="RCIDatabase"
-      connectionString="data source=adminprodsql;initial catalog=RCI;integrated security=True;MultipleActiveResultSets=True;MultiSubnetFailover=true;"
+      connectionString="data source=adminprodsql.gordon.edu;initial catalog=RCITrain;integrated security=True;MultipleActiveResultSets=True;"
```
Comment (EjPlatzer) [inline Phoenix/Web.config:70]: Do you have a plan for this to be updated automatically when pushing to production?

**Review by EjPlatzer — APPROVED** [review]
Assuming you've tested these upgrades, this looks like a good update to me. Thank you for fixing the vulnerable dependencies!

I added a few questions, but none of them are blocking.

File: Phoenix/Web.config (hunk 2 above)
Comment (bennettforkner) [reply Phoenix/Web.config:70]: There is no "push to production" in this project. Everything is done with a Visual Studio publish. Please see Web.Release.Config for the production config.

**Pull request merged**